deploy/olm-catalog/performance-addon-operator/4.5.2/performance-addon-operator.v4.5.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,191 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "performance.openshift.io/v1alpha1",
+          "kind": "PerformanceProfile",
+          "metadata": {
+            "name": "example-performanceprofile"
+          },
+          "spec": {
+            "additionalKernelArgs": [
+              "nmi_watchdog=0",
+              "audit=0",
+              "mce=off",
+              "processor.max_cstate=1",
+              "idle=poll",
+              "intel_idle.max_cstate=0"
+            ],
+            "cpu": {
+              "isolated": "2-3",
+              "reserved": "0-1"
+            },
+            "hugepages": {
+              "defaultHugepagesSize": "1G",
+              "pages": [
+                {
+                  "count": 2,
+                  "node": 0,
+                  "size": "1G"
+                }
+              ]
+            },
+            "nodeSelector": {
+              "node-role.kubernetes.io/performance": ""
+            },
+            "realTimeKernel": {
+              "enabled": true
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    containerImage: REPLACE_IMAGE
```

Comment:
Comparing with 4.5.1 csv I'm missing the following fields here: 
```
    categories: OpenShift Optional
    certified: 'false'
    description: Operator to optimize OpenShift clusters for applications sensitive
      to CPU and network latency.
    repository: https://github.com/openshift-kni/performance-addon-operators
    support: Red Hat
```

Reply:
added by `annotations.json`